frontend/src/metabase/api/parameters.ts

```diff
@@ -0,0 +1,95 @@
+import { getNonVirtualFields } from "metabase-lib/v1/parameters/utils/parameter-fields";
+import { normalizeParameter } from "metabase-lib/v1/parameters/utils/parameter-values";
+import type { Parameter, ParameterValues } from "metabase-types/api";
+
+import { Api, skipToken } from "./api";
+import { idTag } from "./tags";
+
+type NormalizedParameter = ReturnType<typeof normalizeParameter>;
+
+export type LoadParameterValuesParams = {
```

Comment:
Please move types for requests to `metabase-types` and follow naming conventions from other endpoints